### ISSUE TITLE
Ask about epochs and learning rate

### ISSUE THREAD

**Issue by wentaoxandry** [georgesterpu/Sigmedia-AVSR#8]
Hallo @georgesterpu ,

I want ask about your parameters, your code can also implement the experiment, which described in "Lip Reading Sentences in the Wild". I want ask about how many epochs do you have trained and which learning rate you have chosen.

Thank you very much.

**Comment by georgesterpu** [comment]
We only attempted to reproduce the WLAS network, not the entire strategy explained in the article of Chung et al. "Lip Reading Sentences in the Wild". We trained this network on TCD-TIMIT and LRS2 without alternating between modalities or curriculum learning. I can't remember the epoch counter exactly, just that it was much higher than what it normally took for our own method to converge. The predictions and the loss value were not looking promising, so I suspect that either we did not implement the method correctly, or that it is more challenging to make it work. The initial learning rate was `0.001`.
